usaspending_api/search/v2/views/spending_by_category.py

```diff
@@ -100,9 +100,11 @@ def post(self, request: Request) -> Response:
         models.extend(copy.deepcopy(PAGINATION))
 
         # Apply/enforce POST body schema and data validation in request
+        original_filters = request.data.get("filters")
         validated_payload = TinyShield(models).block(request.data)
 
         validated_payload["elasticsearch"] = is_experimental_elasticsearch_api(request)
+        validated_payload["original_filters"] = original_filters
```

Comment:
Why is the original JSON being stored in the validated payload?
Doesn't that undermine the purpose of "validated request JSON"

Reply:
It was a tough call for me, but I thought that linking the history onto the payload object was a little cleaner than two tightly coupled parameters that are derived from each other (not to mention that would introduce conflicts with everyone's ES tickets). The validated payload is still guarenteed to be "valid", in that it no longer has filters that will cause issues; the original_filters variable is more of a tag or description. It is no longer a reflection of the original JSON now that there's a new key, but I feel like this is comparable to something like adding a timestamp to the request.

Reply:
Part of  "validating" the request data is to provide some protection/control of the data in the business logic. This would negate some of that. However, the conversation on the other comment might make this design decision moot.

Reply:
Appears like `validated_payload["original_filters"] = original_filters` is no longer necessary here.

It should also be removed from BaseSpendingByCategoryViewSet and replaced by making original_filters a class attribute